tests/test_DataSet.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import os
+
+import RadClass.DataSet as ds
+import tests.create_file as file
+
+
+def test_init_database():
+    # randomized data to store (smaller than an actual MUSE file)
+    live = np.random.rand(file.timesteps,)
+    timestamps = np.random.rand(file.timesteps,)
+    spectra = np.random.rand(file.timesteps, file.energy_bins)
+
+    file.create_file(live, timestamps, spectra)
+
+    processor = ds.DataSet(file.labels)
```

Comment:
The [internals](https://github.com/CNERG/RadClass/blob/3f35fe0dea9ace0527594624234ed41d501b225a/RadClass/RadClass.py#L72) of `RadClass` call is a processor. I have been using this naming convention because `DataSet` "processes" data. I am open to suggestions, but not opposed to this name.